airflow/providers/apache/hdfs/hooks/webhdfs.py

```diff
@@ -63,39 +63,51 @@ def get_conn(self) -> Any:
         """
         connection = self._find_valid_server()
         if connection is None:
-            raise AirflowWebHDFSHookException("Failed to locate the valid server.")
+            raise AirflowWebHDFSHookException(
```

Comment:
Look like the code formatting has changed.

Reply:
it was changed automatically using the python formatter